frontend-react/src/utils/misc.ts

```diff
@@ -46,38 +46,10 @@ export const checkJson = (
     }
 };
 
-export function includesSpecialChars(text: string): boolean {
-    return /[^a-z\d\s]/i.test(text);
+export function isValidServiceName(text: string): boolean {
+    return /^[a-z\d-_]+$/i.test(text);
```

Comment:
Just glancing at the Organization/Sender/Receivers we have in the production database, this regex should cover all of them.  Important to add underscores because a few of our Senders already them in their names.